src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -466,3 +494,43 @@ def visit_key(self, node, children):
     def visit_quoted_key(self, node, children):
         # leading ! are used to indicate negation. make sure they don't appear.
         return node.match.groups()[0].lstrip("!")
+
+
+def _load_configs():
+    if not CONFIGS_DIR.exists():
+        logger.warning(
+            "Failed to load Fingerprinting Configs, invalid _config_dir: %s",
```

Comment:
Would you want to create a Sentry issue to know that this happened?
Or even a metric?

Otherwise, you would go blind if this was to happen.

Reply:
makinng sure it blows up in CI/Dev if config dir invalid